smart/model.py

```diff
@@ -1825,6 +1892,52 @@ def update_time_dependent_parameters(self):
         # Update time dependent parameters
         for parameter_name, parameter in self.pc.items:
             new_value = None
+            if parameter.type == ParameterType.from_xdmf:
+                # load the time vec from xdmf file
+                tVec = parameter.tVec
+                cur_file = d.HDF5File(self.parent_mesh.mpi_comm, parameter.h5_file, "r")
```

Comment:
If you make a function like I described above, then you might be able to reuse it here